app/app.css

```diff
@@ -45,6 +45,7 @@
 
 .search-result-wrapper{
   width: 100%;
+  margin-bottom: 2%;
```

Comment:
Can we do without this if we use bootstrap styles?

Reply:
this is for the search results repeater to have space between search results items.